content/en/docs/tasks/configure-pod-container/resize-container-resources.md

```diff
@@ -2,264 +2,192 @@
+* If only memory resources are changed, the container is restarted.
+* If *both* CPU and memory resources are changed simultaneously, the container is restarted (due to the memory policy).
 
 ## Limitations
 
-In-place resize of pod resources currently has the following limitations:
+In-place resource resizing currently has the following limitations:
+
+* **Resource Types:** Only CPU and memory resources can be resized.
+* **Memory Decrease:** Memory limits _cannot be decreased_ unless the `resizePolicy` for memory is `RestartContainer`. Memory requests can generally be decreased.
+* **QoS Class:** The Pod's original [Quality of Service (QoS) class](/docs/tasks/configure-pod-container/quality-service-pod/) (Guaranteed, Burstable, BestEffort) is determined at creation and cannot be changed by a resize. The resized resource values must still adhere to the rules of the original QoS class:
+    * *Guaranteed*: Requests must continue to equal limits for both CPU and memory after resizing.
+    * *Burstable*: Requests and limits cannot become equal for *both* CPU and memory simultaneously (as this would change it to Guaranteed).
+    * *BestEffort*: Resource requirements (`requests` or `limits`) cannot be added (as this would change it to Burstable or Guaranteed).
+* **Container Types:** Non-restartable Init Containers and Ephemeral Containers cannot be resized. [Sidecar containers](/docs/concepts/workloads/pods/sidecar-containers/) can be resized.
+* **Resource Removal:** Resource requests and limits cannot be entirely removed once set; they can only be changed to different values.
+* **Operating System:** Windows pods do not support in-place resize.
+* **Node Policies:** Pods managed by [static CPU or Memory manager policies](/docs/tasks/administer-cluster/cpu-management-policies/) cannot be resized in-place.
+* **Swap:** Pods utilizing [swap memory](/docs/concepts/architecture/nodes/#swap-memory) cannot resize memory requests unless the `resizePolicy` for memory is `RestartContainer`.
 
-- Only CPU and memory resources can be changed.
-- Pod QoS Class cannot change. This means that requests must continue to equal limits for Guaranteed
-  pods, Burstable pods cannot set requests and limits to be equal for both CPU & memory, and you
-  cannot add resource requirements to Best Effort pods.
-- Init containers and Ephemeral Containers cannot be resized.
-- Resource requests and limits cannot be removed once set.
-- A container's memory limit may not be reduced below its usage. If a request puts a container in
-  this state, the resize status will remain in `InProgress` until the desired memory limit becomes
-  feasible.
-- Windows pods cannot be resized.
+These restrictions might be relaxed in future Kubernetes versions.
 
+## Example 1: Resizing CPU without Restart
 
-## Create a pod with resource requests and limits
+First, create a Pod designed for in-place CPU resize and restart-required memory resize.
```

Comment:
What namespace is this Pod in? Do you need to create a namespace first?

Reply:
I was thinking they could just use the default namespace, since it's just a single resource being created. Is it standard practice to create a dedicated namespace for all tutorials?